addons/web/static/src/js/views/kanban/kanban_record.js

```diff
@@ -345,14 +345,20 @@ var KanbanRecord = Widget.extend(WidgetAdapterMixin, {
         this.$("widget").each(function () {
             var $field = $(this);
 
+            const attrs = {};
+            for (const attr of $field[0].attributes) {
+                attrs[attr.name] = attr.value;
+            }
+            const options = Object.assign({}, self.options, { attrs });
```

Comment:
@ged-odoo there are subtly diverging `_processWidget` and `_processWidgets` here is that normal?